src/pages/[dao]/[proposal].tsx

```diff
@@ -52,7 +54,17 @@ export default function Proposal() {
     enabled: !!proposal?.id,
     placeholderData: [],
   });
-  const { vote, connected, connect } = useWallet();
+  const { userVote } = useUserVoteByProposalId(
+    Number(params.proposal),
+    "CAZA65HCGNNKGO7P66YNH3RSBVLCOJX5JXYCCUR66MMMBCT7ING4DBJL", //currentGovernor.address,
```

Comment:
use URL param?

Reply:
yes, ill fix this, im currently using the name as url param , gotta change it to address 🤦